eclair-core/src/main/scala/fr/acinq/eclair/CltvExpiry.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair
+
+/**
+ * Created by t-bast on 21/08/2019.
+ */
+
+/**
+ * Bitcoin scripts (in particular HTLCs) need an absolute block expiry (greater than the current block count) to work
+ * with OP_CLTV.
+ *
+ * @param get the absolute cltv expiry value (current block count + some delta).
+ */
+case class CltvExpiry(get: Long) {
```

Comment:
```suggestion
case class CltvExpiry(private val underlying: Long) {
```

And add a `def toLong: Long` getter